ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -1884,6 +1884,9 @@ static int zend_jit_leave_throw_stub(dasm_State **Dst)
 		|5:
 		|	// opline = EG(exception_op);
 		|	LOAD_IP_ADDR_ZTS executor_globals, exception_op
+		||	if (ZEND_OBSERVER_ENABLED) {
```

Comment:
I would recommend to remove the ``if (ZEND_OBSERVER_ENABLED) {`` condition.
We already have the corresponding unconditional ``STORE`` in master.

Reply:
Will do :-)